io.github.Predidit.Kazumi.metainfo.xml

```diff
@@ -0,0 +1,396 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop-application">
+  <!--Created with jdAppStreamEdit 9.1-->
+  <id>io.github.Predidit.Kazumi</id>
+  <name>Kazumi</name>
+  <summary>Watch Animes online with danmaku support</summary>
+  <summary xml:lang="zh_CN">一款好用的追番软件</summary>
+  <developer id="com.predidit">
```

Comment:
`predidit.com` doesn't exist.

Reply:
fixed

Reply:
Seems not fixed:

```
resolve predidit.com                  
Resolving [predidit.com] using 127.0.0.1 port 53

Resolver      : 172.217.34.213

Canonical name: predidit.com.

IPv4 addresses: -
IPv6 addresses: -

Name servers  : name does not exist
DNSSEC signed : no
Mail servers  : no mail servers found

HTTPS alias   : -
HTTPS info    : -

Host info     : -
TXT records   : -
```

Reply:
You can use `io.github.Predidit` also there.

Reply:
Ok looks like it was already changed to that